src/commons/util.js

```diff
@@ -191,19 +191,33 @@ export function timeout(ms, callback) {
   return setTimeout(callback, ms);
 }
 
+import crypto from "crypto";
+
 // min inclusive, max exclusive
 export function rand(min, max) {
-  return Math.floor(Math.random() * (max - min)) + min;
+  const range = max - min;
+  const bytesNeeded = Math.ceil(Math.log2(range) / 8);
+  const randomBytes = crypto.randomBytes(bytesNeeded);
```

Comment:
`Math.random` because it is the fastest random number generator in JS. These IDs it generates are merely used for debugging purposes and so, it doesn't matter if these are crytpographically secure.